resources/views/site/shared/_featuredRelated.blade.php

```diff
@@ -17,9 +17,30 @@
     if (($behavior ?? '') === 'relatedSidebar') {
         $listingVariation .= ' m-listing--dynamic';
     }
+
+    // Get featured related items
+    $isFeatured = false;
+    $featuredRelated = collect($item->getFeaturedRelated())->filter()->map(function ($featuredRelated) {
+        return $featuredRelated;
+    });
```

Comment:
I don't quite understand what's happening here. The `filter()` will remove the `null` items, but the `map()` seems to just return every item without modification.

Reply:
I see you removed the `filter()`, but what's the purpose of the `map()` that just returns the item?

Reply:
I think you can use the Laravel Collections `[pluck](https://laravel.com/docs/10.x/collections#method-pluck)` method to do this more efficiently. 

```
collect($item->getFeaturedRelated())->pluck('item');
```